CMakeLists.txt

```diff
@@ -127,10 +127,16 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_GLIBCXX_USE_CXX11_ABI=${NGRAPH_TF_CXX11_ABI}")
 endif()
 
```

Comment:
note to self:
right now we create .so only
after this change we will create .a AND .so. python will use .so and if cpp apps can use .a (and possibly .so). CI gtest is shifting from .so to .a, so .so <-> python and .a <-> cpp